Project/Gem/CMakeLists.txt

```diff
@@ -77,6 +84,9 @@ ly_add_target(
             Gem::${gem_name}.Private.Object
             AZ::AzCore
             Gem::ROS2.Static
+            AZ::AzGameFramework
+            AZ::AzFramework
+            AZ::AzToolsFramework
```

Comment:
Tools Framework dependency on Gem target is not correct, as it prevents building launcher only. AzToolsFramework is Editor dependency and should only be included as dependency for the primary Gem target.

See CMakeLists.txt of the ROS 2 Gem.

Reply:
Also, it does not seem to be used. Please review dependencies.